### ISSUE TITLE
Improving the subscription UX

### ISSUE THREAD

**Issue by marienfressinaud** [FreshRSS/FreshRSS#2014]
The experience when adding a feed is a bit chaotic and could be improved a lot.

First, the form to add a feed isn't too hard to find but could be more visible (categories are taking a lot of place and can "distract" user who can miss the smaller "add" form).

The dropbox between the input and the submit button is very confusing. What happens when we click on it? Why do we need an HTTP authentication?

Once we subscribed to the feed, what is this strange panel with a lot of options? Did I successfully subscribe to the feed? But what is this "remove" button? If I click on "submit", nothing happens :/ And how do I close this panel? I just want to read articles or to add more feeds.

**Comment by math-GH** [comment]
Right now (1.20.0) it looks like this:

![grafik](https://user-images.githubusercontent.com/1645099/188285781-c9ac1263-e5d9-41fb-bc29-9603534408f5.png)

I guess since 2018 it happened a lot of changes here.

Any thoughts about the actual UX?

**Comment by Frenzie** [comment]
I still think it's confusing, if only because when you click + you (almost) never want to add a category, yet it's situated at the top.

**Comment by math-GH** [comment]
Would it make sense to have this forms as slider in the "subscription management" page, where you have an overview of feeds and categories?

**Comment by Frenzie** [comment]
I'm not entirely sure what you mean, but it was very purposefully made available from + without having to perform many clicks. I'd just swap add a feed and add a category.

**Comment by marienfressinaud** [comment]
I agree with @Frenzie, swapping "add a feed" and "add a category" would be better.

I would also suggest to redirect to the main view after adding a feed so the user can read it. Or maybe just stay on the adding page so he can add more feeds?
